index.rst

```diff
@@ -77,6 +77,7 @@ For a complete list of our projects, check out `Open Data Kit on Github <https:/
   form-update
   pyxform
   validate
+  screenshots
```

Comment:
I think this guide should probably be in the contributor's sections, not the form building section.